test/listen.test.js

```diff
@@ -199,6 +200,10 @@ if (os.platform() !== 'win32') {
       t.error(err)
       t.equal(sockFile, fastify.server.address())
       t.equal(address, sockFile)
+
+      fastify.close(() => {
+        t.equal(fs.existsSync(sockFile), false)
+      })
```

Comment:
Would you mind including this in https://github.com/fastify/fastify/pull/1740/files#diff-453c0218a85edef172564771cab22764R191 instead?

Reply:
I removed the fastify.close added above and replaced the `fastify` parameter inside the `.bind` function with `t.equal`. It seems to work well. I tried changing the flag from `false` to `true`, or changing the `plan` value from `4` to `3`, and the tests seem to respond correctly. I'm not sure this is the right approach. What do you think about it?